### DISCUSSION TITLE
Use creator in extra field when there's no creator in the usual places?

### DISCUSSION THREAD

**Discussion by Jmuccigr** [retorquere/zotero-better-bibtex#1871] [Q&A] [answered]
I've [just discovered](https://forums.zotero.org/discussion/comment/386724#Comment_386724) that Zotero has no "editor" option for the creator of a web page, but the usual "cheat" of putting one into the extra field works fine. However BBT seems to want nothing to do with using that field to create a cite key. I put `[extra=editor:lower][year][0] | [auth:(unknown)][year][0]` in the format, and it ignores the editor field. If I use another name for it, like "blah" or tex.editor, it works fine. Of course that also means that Zotero no longer recognizes it as a creator and so it won't end up in the exported citations.

There's also the issue of the format Zotero needs for names there, that is "Last || First", which I suppose I could handle with regex so BBT doesn't process the whole thing.

Am I missing the proper way to do this?

**Comment by retorquere** [comment, marked as answer]
This isn't currently implemented, but if you join me in #1873 I can fix that.
